controllers/apps/v1beta3/emqx_handler.go

```diff
@@ -957,3 +898,16 @@ func isPluginInitialized(instance appsv1beta3.Emqx) bool {
 	}
 	return false
 }
+
+func generateEmqxEnvKey(key string) string {
+	return fmt.Sprintf("EMQX_%s", strings.ToUpper(strings.ReplaceAll(key, ".", "__")))
+}
+
+func getManagerField(emqx appsv1beta3.Emqx) (string, string, string) {
```

Comment:
Maybe we don't need this function, because EMQX never equal nil, so if we need username and password, we just use `emqx.GetUsername()` and `emqx.GetUsername()`

Reply:
these fields need to be referenced in many places, so I put them in a method

Reply:
Yes, but it just call Get function, and return value, it has no logic, so is this function necessary?

Reply:
this is indeed a get method, and if we need other fields, we can also use this method to return without adding code

Reply:
But we don't need other fields for now, If we need it in the future, we'll just create the function at that time